Editor/CoreAPI/StateManager.cs

```diff
@@ -20,15 +21,20 @@ public class StateManager
         public string SelectedProfile
         {
             get => _selectedProfile;
-            private set => SetProfile(value);
+            set => SetProfile(value);
         }
 
+        public IReadOnlyList<string> AllProfiles => CoreApi.ListCredentialsProfiles().Profiles.ToList();
+        
+        public bool IsBootstrapped { get; set; }
+
         public StateManager(CoreApi coreApi)
         {
             CoreApi = coreApi;
             AmazonGameLiftClientFactory = new AmazonGameLiftClientFactory(coreApi);
 
             SetProfile(coreApi.GetSetting(SettingsKeys.CurrentProfileName).Value);
+            IsBootstrapped = coreApi.GetSetting(SettingsKeys.IsBootstrapped).Value == "true";
```

Comment:
This matter is a bigger task that I thought, I am currently working on implementing a User profile to save this sort of data to. This will be done in a separate PR.

Reply:
For future reference, this is the follow-up PR: https://github.com/aws/amazon-gamelift-plugin-unity/pull/108